libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -53,6 +53,11 @@ void AP_Mount_Backend::update()
         mnt_target.rate_rads.pitch = 0;
         mnt_target.rate_rads.yaw = 0;
     }
+    // location exists for GPS_POINT mode
+    Location current_loc;
+    if (get_mode() == MAV_MOUNT_MODE_GPS_POINT && !AP::ahrs().get_location(current_loc)) {
```

Comment:
Is this the only mode where the warning is relevant?  I'd think "point home" would be similarly affected.  Probably want a `switch` and evaluate each of the modes in there.

Reply:
Added HOME, Randy did not think that SYSID needed it

Reply:
@Hwurzburg thanks very much for all this work, it's looking pretty good.

Sorry, I was probably unclear.  I think that the three modes that could use this check are MAV_MOUNT_MODE_GPS_POINT, MAV_MOUNT_MODE_HOME_LOCATION and MAV_MOUNT_MODE_SYSID_TARGET.  SysId is so rarely used that it hardly matters but it might as well get the check.

I agree with PeterB that we should use a switch to force any future developer who changes to the enum (e.g. adds a new submodes) to consider whether this check should apply to the new mode or not